assets/js/analytics/base.js

```diff
@@ -290,103 +290,127 @@ export class BaseAnalyticsView extends Component {
-                    <CorrelationTableRow key={key} object={key} />
-                  ))}
-                </tbody>
-              </table>
-            </div>
-          </div>
+        <div className="chart-wrapper">
+          <Bar
+            data={this.state.selectedUserActivitiesCorrelationsChart}
+            options={{
+              maintainAspectRatio: true
+            }}
+          />
         </div>
       </div>
     );
   }
 
-  renderSupplementsCorrelations() {
+  renderUserActivitiesData() {
     return (
-      <div className="card-columns cols-2">
+      <div className="float">
         <div className="card">
-          <div className="card-header analytics-text-box-label">
-            {this.supplementsCorrelationsChartLabel}
-          </div>
-          <div className="chart-wrapper">
-            <Bar
-              data={this.state.supplementsCorrelationsChart}
-              options={{
-                maintainAspectRatio: true
-              }}
-            />
+          <Nav tabs>
+            {this.renderActivitiesCorrelationsSelectionTab(
+              POSITIVELY_CORRELATED_LABEL
+            )}
+            {this.renderActivitiesCorrelationsSelectionTab(
+              NEGATIVELY_CORRELATED_LABEL
+            )}
+            {this.renderActivitiesCorrelationsSelectionTab(
+              NOT_CORRELATED_LABEL
+            )}
+          </Nav>
+          <div className="card-block">
+            <table className="table">
+              <thead>
+                <tr>
+                  <th>Activity</th>
+                  <th>Correlation</th>
+                </tr>
+              </thead>
+              <tbody>
+                {this.state.selectedUserActivitiesCorrelations.map(key => (
+                  <CorrelationTableRow key={key} object={key} />
+                ))}
+              </tbody>
+            </table>
           </div>
         </div>
-        <div className="float">
-          <div className="card">
-            <Nav tabs>
-              {this.renderSupplementsCorrelationsSelectionTab(
-                POSITIVELY_CORRELATED_LABEL
-              )}
-              {this.renderSupplementsCorrelationsSelectionTab(
-                NEGATIVELY_CORRELATED_LABEL
-              )}
-              {this.renderSupplementsCorrelationsSelectionTab(
-                NOT_CORRELATED_LABEL
-              )}
-            </Nav>
-            <div className="card-block">
-              <table className="table">
-                <thead>
-                  <tr>
-                    <th>Supplement</th>
-                    <th>Correlation</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {this.state.selectedSupplementsCorrelations.map(key => (
-                    <CorrelationTableRow key={key} object={key} />
-                  ))}
-                </tbody>
-              </table>
-            </div>
+      </div>
+    );
+  }
+
+  renderUserActivitiesCorrelations() {
```

Comment:
You could probably refactor this more, but this entire file has to be refactored when you're done, so it's not worth it.